app/pathogen/arbovirus/dashboard/(map)/MapAndFilters.tsx

```diff
@@ -1,14 +1,15 @@
 "use client";
```

Comment:
This is really awesome and interesting. I remember using a react mapbox tool in the past (but I do not think it was this) and deciding against it because it had a bunch of issues. Looking at the documentation for this is p impressive and having it all be React-like would be great